Update data.en.md (#277)
OSGeo/grass-website: content/download/data.en.md

  <li>
   <span class="mwl"><a href="/grass-stable/manuals/ " target="_blank"> Climate </a></span>
   <a href="/sampledata/north_carolina/nc_climate_spm_2000_2012.zip" class="inl btn btn-primary" target="_blank">Download ZIP (690 MB)</a>

>    <p>North Carolina mapset with climatic data time series (nc_climate_spm_2000_2012). To unzip in North Carolina location of the NC full or basic dataset  - see also the related <a href="http://ncsu-geoforall-lab.github.io/grass-temporal-workshop/">tutorial</a></p>

Reviewer: ```suggestion
   <p>North Carolina mapset with climatic data time series (nc_climate_spm_2000_2012). Unzip within North Carolina location, either full or basis datasets. See also the related <a href="http://ncsu-geoforall-lab.github.io/grass-temporal-workshop/">tutorial</a></p>
```
